includes/Register_Font.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Class Register_Font.
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories;
+
+/**
+ * Class Register_Font
+ *
+ * @package Google\Web_Stories
+ */
+class Register_Font {
+	/**
+	 * Font Style handle.
+	 *
+	 * @var string
+	 */
+	const STYLE_HANDLE = 'google-fonts';
+
+	/**
+	 * Prefix handle.
+	 *
+	 * @param string $handle Style handle used to prefix.
+	 *
+	 * @return string
+	 */
+	public function get_handle( $handle ) {
+		return $handle . '-' . self::STYLE_HANDLE;
```

Comment:
This could be `self::STYLE_HANDLE . '-' . $handle;` instead to keep the prefix consistent.

Reply:
No, handle is the prefix. So in two examples it is used, it is `edit-story-google-fonts` and `stories-dashboard-google-fonts`.

Reply:
I can see that it's currently like this. However, why not use the plugin slug as the prefix as generally used and recommended in case of WP?

Looking that `web-stories-fonts` is also under the Expected behavior of the original issue.

<img width="860" alt="Screenshot 2021-04-22 at 15 36 44" src="https://user-images.githubusercontent.com/3294597/115715189-32494e00-a378-11eb-9a4b-196890a9d51b.png">

Reply:
There is also this in the ticket. 
> Bonus: stylesheet registration is de-duped. Might require higher estimate.

Basically, the style is registered twice with the same handle. This is isn't great. So now it prefixed with the handle of the parent style, so it is not a duplicate.

Reply:
Yes, but why use `google-fonts` instead of `web-stories-fonts` for the `self::STYLE_HANDLE`? It would still be de-duped, no?